src/app/design/guidelines/back-button/index.html

```diff
@@ -0,0 +1,165 @@
+  </p>
+  <p>
+    Behaviors vary based on different browser and screen reader combinations, but SKY UX applications use the combination listed above.
+  </p>
+
+  <stache-page-anchor>
+    Tabs and vertical tabs as site level navigation
+  </stache-page-anchor>
+
+  <p>
+    If you use tabs as page navigation, each tab should be a page in the browser history, and users should be able to use the back button to return to the previous view. Tabs contain all of a page's content and can't be dismissed.
+  </p>
+
+  <div class="screenshot screenshot-info">
+    <img src="~/assets/img/guidelines/back-button/tabs-backbutton.png" alt="Tabs as page navigation" style="width: 100%" />
+  </div>
+
+  <div class="sky-subsection-heading" style="margin-top: 20px;">Focus</div>
+
+  <p>
+    Based on guidance from <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel">W3C's WAI-ARIA Authoring Practices for tabs</a>, maintain focus on activated tabs.
+  </p>
+
+  <div class="sky-subsection-heading">New page announcement for screenreaders</div>
+
+  <p>
+    Titles are announced as described in <a stacheRouterLink="." fragment="new-page-behavior">New page annoucement for screenreaders</a>.
+  </p>
+
+  <stache-page-anchor>
+    Tabs as containers for viewing items
+  </stache-page-anchor>
+
+  <p>
+    If tabs contain removable items such as lists or reports, don't include each tab in the browser history.
+  </p>
+
+	<div class="screenshot screenshot-info"> 
+    <img src="~/assets/img/guidelines/back-button/tabcontainers-backbutton.png" alt="Tabs as page navigation" />
+	</div>
+
+<br>
+
+  <sky-row>
+    <sky-column>
+      <img class="screenshot screenshot-info" src="~/assets/img/guidelines/back-button/verticaltabs-backbutton.png" alt="Vertical tabs as navigation" />
+    </sky-column>
+  </sky-row>
+
+  <div class="sky-subsection-heading" style="margin-top: 20px;">Focus</div>
+
+  <p>
+    Based on guidance from <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel">WAI-ARIA Authoring Practices for tabs</a>, maintain focus on activated tabs.
+  </p>
+
+  <div class="sky-subsection-heading">New page announcement for screenreaders</div>
+
+  <p>
+    This option doesn't require new page announcements. The selected tab receives an updated <sky-code>aria-selected="true"</sky-code> property which is communicated to assistive technologies.
+  </p>
+
+  <stache-page-anchor>
+    Modals (all sizes)
+  </stache-page-anchor>
+
+  <p>
+    When users open a modal from a SKY UX SPA, they should be able to use the back button to close the modal and return to the original context.
+  </p>
+
+  <div class="sky-subsection-heading">Avoid loss of work</div>
+
+  <p>
+    If a user selects the back button on a form with unsaved data, the SKY UX application asks the user if they want to close the form without saving their data or work using the confirmation dialog.
```

Comment:
I'd change "a user selects" to "users select" and "the user" to "they." Also, we say "a form" and "the form," but I think this should be "a modal" and "the modal." ... I'd change "SKY UX application" to just "SPA." ... Also, as written, it sounds like the users are saving the work with the confirmation dialog. I'd move this up in the sentence: "If users select the back button on a modal with unsaved data, the SPA displays a dialog to confirm that they want to close the modal without saving their work."